.github/workflows/fast-tests.yml

```diff
@@ -44,6 +44,10 @@ jobs:
     name: Run Db Versions Minimal Tests
     uses: ./.github/workflows/test-db-versions-minimal.yml
 
+  db-version-minimal-gpu-tests:
```

Comment:
This should not be a fast test, it should only run when slow tests run, because it is about 10 times as expensive as on a normal runner

Reply:
True. To my excuse I used the fast tests for faster iterations, but of course, the final implementation should run it in the slow tests

Reply:
Done